go/vt/vtctl/grpcvtctlclient/client.go

```diff
@@ -49,8 +50,15 @@ func gRPCVtctlClientFactory(addr string, dialTimeout time.Duration) (vtctlclient
 	if err != nil {
 		return nil, err
 	}
+	opts := []grpc.DialOption{opt}
+	opts = append(opts, grpc.WithTimeout(dialTimeout))
```

Comment:
Minor nit -- I think you could consolidate this to
`opts := []grpc.DialOption{opt, grpc.WithTimeout(dialTimeout))}`